cluster/manifests/coredns/configmap.yaml

```diff
@@ -11,10 +11,18 @@ data:
         errors
         health
         kubernetes cluster.local in-addr.arpa ip6.arpa {
+            resyncperiod 1m
```

Comment:
@njuettner Should we do this in a separate PR or move into the `if`? This makes this PR essentially contain two changes (1 guarded by a config item and this one). The default sync period is `5m`, right?

Reply:
Correct, the default would be 5 minutes. I'll add an `if`.

Reply:
Ok. I'm always unsure about this stuff. Just want to mention that's also possible to use a config item and set its default value to `5m` to avoid the `if` or to leave it out entirely for now.